js/app/controllers/content-product.ng

```diff
@@ -209,6 +209,158 @@ Import-GcSqlInstance "my-instance-name" "C:\Users\User\file.csv" `
+                     -Subscription "push-subscription" `
+                     -PushEndpoint "http://www.example.com"
+</pre>
+    <p>
+      To pull messages from a subscription, the <cmdlet-ref name="Get-GcpsMessage"></cmdlet-ref> cmdlet can
+      be used. By default, the cmdlet will block until at least one message is retrieved. To prevent blocking,
+      the switch <code>-ReturnImmediately</code> can be used. The cmdlet can also automatically send
+      an acknowledgement for every retrieved message if the switch <code>-AutoAck</code> is used. If not,
+      you will have to use the <cmdlet-ref name="Send-GcpsAck"></cmdlet-ref> cmdlet to send the acknowledgement.
+      Unacknowledged messages will become available again for pulling after the acknowledgement deadline of the message expires.
+    </p>
+    <pre>
+# Pulls messages from subscription "my-subscription" and sends out acknowledgement automatically.
+Get-GcpsMessage -Subscription "--subscription" -AutoAck
+
+# Pulls messages from subscription "my-subscription" and sends out acknowledgement with Send-GcpsAck.
+$messages = Get-GcpsMessage -Subscription "--subscription"
+Send-GcpsAck -InputObject $messages</pre>
+  </div>
+
+  <!-- Google Cloud Logging -->
+  <div ng-switch-when="google-cloud-logging">
+    <p>
+      Stackdriver Logging allows you to store, search, analyze, monitor and alert on log data
+      and events from Google Cloud Platform and Amazon Web Services.
+    </p>
+    <h2>Log and log entries</h2>
+    <p>
+      A log is a named collection of log entries within the project. A log entry records status or an event.
+      The entry might be created by GCP services, AWS services, third party applications, or your own applications.
+      The "message" the log entry carries is called the payload, and it can be a simple string or structured data.
+      Each log entry indicates where it came from by including the name of a monitored resource.
+    </p>
+    <p>
+      The cmdlet <cmdlet-ref name="New-GcLogEntry"></cmdlet-ref> can be used to create a log entry.
+      You will have to specify the log that the entry belongs to (if the log does not exist, it will
+      be created). To associate the log with a monitored resource, you can use the -MonitoredResource parameter.
+      By default, the log entry is associated with the "global" resource. To create a monitored resource,
+      use the <cmdlet-ref name="New-GcLogMonitoredResource"></cmdlet-ref> cmdlet.
+  </p>
+    <pre>
+# Creates a log entry in the log "my-log".
+New-GcLogEntry -LogName "my-log" -TextPayload "This is a log."
+
+# Creates a log entry associated with Cloud SQL monitored resource.
+$resource = New-GcLogMonitoredResource -ResourceType "cloudsql_database" `
+                                       -Labels @{"project_id" = "my-project";
+                                                 "database_id" = "id"}
+New-GcLogEntry -LogName "my-log" `
+               -TextPayload "Log" `
```

Comment:
I would change this payload to be a little larger to not be confusing what needs to be here